cfgov/jinja2/v1/_includes/organisms/content-banner.html

```diff
@@ -0,0 +1,33 @@
+{# ==========================================================================
+
+   Banner
+
+   ==========================================================================
+
+   Description:
+
+   Create an banner when given:
+
+    value:         				An object used to customize the markup.
+
+   	value.border_top:       	Whether banner should have a top border.
+
+   	value.border_bottom:    	Whether banner should have a bottom border.
+
+   	value.content:          	Banner content
+
+   	value.additional_classes: 	Additional banner classes
+
+   ========================================================================== #}
+	<div class="content_banner 
```

Comment:
Moved to `/templates` and updated documentation in https://github.com/cfpb/cfgov-refresh/pull/2932/commits/df690923fca3e24cdfd7bc3deb33ced2fb51892a